src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -196,6 +201,15 @@ public String deleteAccount(UUID userId) {
         return String.format("User with ID %s has been deleted", userToDelete.getUserId());
     }
 
+    public String deleteAllAccountsWithEmailPrefix(String prefix) {
+        List<UUID> userIds = accountFilteringService.findAllAccountsExceptThirdParty(
+            PageRequest.of(0, 25), prefix, "", Collections.emptyList(), Collections.emptyList(), ""
```

Comment:
Could we highlight the limit of 25 in the java doc of the controller method to make it clear in case we ever go over this in the future

Reply:
I have now made the endpoint delete all accounts rather than limiting to the first batch of 25 accounts